omega/omega-transaction/src/main/java/org/apache/servicecomb/pack/omega/transaction/TransactionContextUtils.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.servicecomb.pack.omega.transaction;
+
+import org.apache.servicecomb.pack.omega.context.OmegaContext;
+import org.apache.servicecomb.pack.omega.context.TransactionContext;
+import org.apache.servicecomb.pack.omega.context.TransactionContextProperties;
+import org.slf4j.Logger;
+
+public abstract class TransactionContextUtils {
```

Comment:
Static method is hard for extension, my suggestion is letting the TransactionAspect, TccParticipatorAspect extends the class of TransactionContextHelper.